third_party/airflow/armada/operators/armada_deferrable.py

```diff
@@ -100,6 +103,10 @@ def __init__(
         super().__init__(**kwargs)
         self.name = name
         self.armada_channel_args = GrpcChannelArguments(**armada_channel_args)
+
+        if "options" not in job_service_channel_args:
```

Comment:
Should this code also be in armada.py?

Reply:
The interfaces of `ArmadaOperator` and `ArmadaDeferrableOperator` are, unforunately, different. `ArmadaOperator` takes already constructed clients that have already instantiated channels attached. `ArmadaDeferrableOperator` has to construct clients and channels on the fly due to its deferrable nature.